lib/import.php

```diff
@@ -76,7 +77,7 @@ public function payload( WordPress_GitHub_Sync_Payload $payload ) {
 	 * @return string|WP_Error
 	 */
 	public function master() {
-		return $this->commit( $this->app->api()->fetch()->master() );
+		return $this->commit( $this->app->api()->fetch()->master(), null );
```

Comment:
Instead of passing in `null` here, we should not pass in anything and update the default value for this method.